{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-25T07:30:00.000+0000",
      "latitude": 50.93988379526646,
      "estimatedRestoration": "2022-04-28T15:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '207012'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/207012",
      "reference": "207012",
      "name": "SO32 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO32 2BP",
        "SO32 2BQ",
        "SO32 2BR",
        "SO32 2BS",
        "SO32 2EH",
        "SO32 2EJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.258366284153947,
          50.93988379526646
        ],
        "type": "Point"
      },
      "UUID": "9fba013e-9185-41a4-8985-8ca9ee57f5cb",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.258366284153947,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-25T19:28:00.000+0000",
      "latitude": 51.21800364797035,
      "estimatedRestoration": "2022-04-26T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2520'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2520",
      "reference": "JS2520",
      "engineerOnSiteTime": "2022-04-25T21:00:00.000+0000",
      "name": "SP10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP10 5AT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.4580420700456025,
          51.21800364797035
        ],
        "type": "Point"
      },
      "UUID": "335aa4bc-f945-46e5-a95c-a8b1916de34e",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4580420700456025,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T02:01:00.000+0000",
      "latitude": 50.95720147431817,
      "estimatedRestoration": "2022-04-26T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2573'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2573",
      "reference": "JS2573",
      "engineerOnSiteTime": "2022-04-26T02:55:00.000+0000",
      "name": "DT9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "DT9 4DY",
        "DT9 4DZ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.5143720913652747,
          50.95720147431817
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-26T03:00:00.000+0000",
      "UUID": "fd782deb-f0a7-4635-ab4a-e3d33484722e",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.5143720913652747,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T03:33:00.000+0000",
      "latitude": 50.76538612251468,
      "estimatedRestoration": "2022-04-26T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2591'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2591",
      "reference": "JS2591",
      "engineerOnSiteTime": "2022-04-26T04:29:00.000+0000",
      "name": "BH11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH11 9EN",
        "BH11 9ES",
        "BH11 9ET",
        "BH11 9EU",
        "BH11 9EW",
        "BH11 9EX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.911894297009315,
          50.76538612251468
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-26T04:30:00.000+0000",
      "UUID": "1607efc1-ed92-4c19-93b0-4925b5473991",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.911894297009315,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T05:13:00.000+0000",
      "latitude": 50.76538612251468,
      "estimatedRestoration": "2022-04-26T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2613'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2613",
      "reference": "JS2613",
      "name": "BH11 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH11 9EN",
        "BH11 9ES",
        "BH11 9ET",
        "BH11 9EU",
        "BH11 9EW",
        "BH11 9EX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.911894297009315,
          50.76538612251468
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-26T07:00:00.000+0000",
      "UUID": "d4c1fc17-b8a8-444e-9a74-fe8afb98e828",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.911894297009315,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T06:00:00.000+0000",
      "latitude": 50.987710367676506,
      "estimatedRestoration": "2022-04-26T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206828'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206828",
      "reference": "206828",
      "name": "SP5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP5 2BL",
        "SP5 2BQ",
        "SP5 2HA",
        "SP5 2HU",
        "SP5 2HX",
        "SP5 2HY",
        "SP5 2HZ",
        "SP5 2JA",
        "SP5 2JB",
        "SP5 2JD",
        "SP5 2JE",
        "SP5 2JF",
        "SP5 2JG",
        "SP5 2JH",
        "SP5 2JJ",
        "SP5 2JL",
        "SP5 2JN",
        "SP5 2JQ",
        "SP5 2JU",
        "SP5 2JZ",
        "SP5 2LX",
        "SP5 2LY",
        "SP5 2LZ",
        "SP5 2NA",
        "SP5 2NB",
        "SP5 2PY"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7203515335328545,
          50.987710367676506
        ],
        "type": "Point"
      },
      "UUID": "98d542e5-e4e4-463b-b2f6-8198f99e6f80",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7203515335328545,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T05:31:00.000+0000",
      "latitude": 51.449379085754465,
      "estimatedRestoration": "2022-04-26T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2618'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2618",
      "reference": "JS2618",
      "name": "RG5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG5 4HH"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.9012863477697319,
          51.449379085754465
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-26T06:45:00.000+0000",
      "UUID": "83b16607-9969-4738-a16c-765fac5bf904",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9012863477697319,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-25T06:00:00.000+0000",
      "latitude": 56.59484562207691,
      "estimatedRestoration": "2022-04-29T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206904'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206904",
      "reference": "206904",
      "name": "DD11 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "DD11 2QR",
        "DD11 4RW",
        "DD11 5RG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.6261701718912582,
          56.59484562207691
        ],
        "type": "Point"
      },
      "UUID": "e856f5ac-52d2-4e6b-b6af-5d5dd1483949",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.6261701718912582,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-26T02:41:00.000+0000",
      "latitude": 57.00647232770239,
      "estimatedRestoration": "2022-04-26T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS2578'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS2578",
      "reference": "JS2578",
      "name": "PH41 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH41 4QF",
        "PH41 4QG",
        "PH41 4QH",
        "PH41 4QP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -5.821688109319294,
          57.00647232770239
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-26T03:45:00.000+0000",
      "UUID": "bfee866d-1d22-46af-9a97-aaf9bc4a6d11",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.821688109319294,
      "resolved": false
    }
  ]
}
